units/systemd-coredump@.service.in

```diff
@@ -19,9 +19,19 @@ Before=shutdown.target
 ExecStart=-@rootlibexecdir@/systemd-coredump
 Nice=9
 OOMScoreAdjust=500
+RuntimeMaxSec=5min
```

Comment:
I don't think that's good. People do have occasional chrome coredumps which can be multi-gig and might take longer than that to process. RuntimeMaxSec should be used a safety net, i.e. it should be higher than any expected usage. To limit coredump from compressing to big cores, ulimit and maximum process size settings internal to systemd-coredump should be preferred.